src/io/iter_sparse_batchloader.h

```diff
@@ -100,15 +100,15 @@ class SparseBatchLoader : public BatchLoader, public SparseIIterator<TBlobBatch>
     return false;
   }
 
-  virtual const TBlobBatch &Value(void) const {
+  virtual const TBlobBatch& Value(void) const {
```

Comment:
I think both styles are fine: https://google.github.io/styleguide/cppguide.html#Pointer_and_Reference_Expressions

No need to change this